ppr-ui/src/components/tables/common/SortingIcon.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <span>
+    <v-icon
+      v-if="sortAsc"
+      small
+      color="black"
+      @click="emitSort(true)"
+      data-test-id="down-arrow-icon"
+    >
+      mdi-arrow-down
+    </v-icon>
+    <v-icon
+      v-else
+      small
+      color="black"
+      @click="emitSort(false)"
+      data-test-id="up-arrow-icon"
+    >
+      mdi-arrow-up
+    </v-icon>
+  </span>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive, toRefs } from '@vue/composition-api'
+
+export default defineComponent({
+  name: 'SortingIcon',
+  emits: ['sortEvent'],
+  components: {},
+  props: {
+    sortAsc: {
+      type: Boolean,
+      default: false
+    }
+  },
+  setup (props, context) {
+    const localState = reactive({})
+    const emitSort = (val) => context.emit('sortEvent', val)
```

Comment:
Maybe for clarity, instead of `val` we can pass something like `isAscending`. Just an idea to give more context without looking up what `val` is.

Reply:
Easy peasy 👍